### PR TITLE
#10608 update column length

### PR DESCRIPTION
**Pull request by erickgonzalez**

### CODE REVIEW

File: src/com/dotmarketing/startup/runonce/Task03605FixMSSQLMissingConstraints.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ * passing query parameters as UTF-8, which forces a whole table scan for all
+ * tables: inode, tree, identifier tables, etc. This makes SQL Server operations
+ * slow. In its current state, it is incapable of serving UTF-8 content.
+ * 
+ * @author Jose Castro
+ * @version 3.7
+ * @since Nov 10, 2016
+ *
+ */
+public class Task03605FixMSSQLMissingConstraints extends AbstractJDBCStartupTask {
```
Comment (oarrietadotcms) [inline src/com/dotmarketing/startup/runonce/Task03605FixMSSQLMissingConstraints.java]: I would change the number of the Task to Task03710...

Comment (jcastro-dotcms) [inline src/com/dotmarketing/startup/runonce/Task03605FixMSSQLMissingConstraints.java]: Also, the Javadoc should be updated.

Comment (erickgonzalez) [inline src/com/dotmarketing/startup/runonce/Task03605FixMSSQLMissingConstraints.java]: Javadoc updated. In 4.0 there is already a Task named 03710, so making that change would imply renaming lots of upgrade tasks for 4.0 and above.

**Comment by oarrietadotcms** [comment]
@jcastro-dotcms could you please review this PR?

File: src-conf/mssql.sql [hunk 2]
```diff
@@ -1327,12 +1327,12 @@ create table workflow_task (
...
    created_by NVARCHAR(255) null,
-   assigned_to NVARCHAR(255) null,
+   assigned_to NVARCHAR(36) null,
    belongs_to NVARCHAR(255) null,
    title NVARCHAR(255) null,
    description NVARCHAR(MAX) null,
-   status NVARCHAR(255) null,
-   webasset NVARCHAR(255) null,
+   status NVARCHAR(36) null,
+   webasset NVARCHAR(36) null,
```
Comment (jcastro-dotcms) [inline src-conf/mssql.sql]: If the size of these fields is changed here, shouldn't it be changed in all other scripts as well (Oracle, Postgres, etc.)? Is this something specific to a customer's DB?

Comment (erickgonzalez) [inline src-conf/mssql.sql]: Only MSSQL complain about the issue, but it can be changed in all others scripts. What do you think @oarrietadotcms ?

Comment (jcastro-dotcms) [inline src-conf/mssql.sql]: I'm a little concerned because the default NVARCHAR size for all other scripts is 255

Comment (erickgonzalez) [inline src-conf/mssql.sql]: Yeah I know, but the others DB doesn't seem to care about the diff between the length. E.g: 
Foreign-key constraints:
```
    "fk_workflow_assign" FOREIGN KEY (assigned_to) REFERENCES cms_role(id)
    "fk_workflow_step" FOREIGN KEY (status) REFERENCES workflow_step(id)
    "fk_workflow_task_asset" FOREIGN KEY (webasset) REFERENCES identifier(id)
```

Comment (dsilvam) [inline src-conf/mssql.sql]: @erickgonzalez @jcastro-dotcms this seems ok. I understand what @erickgonzalez is saying. Only MSSQL is complaining about that matched column-length. We should be fine with this.

**Review by dsilvam — APPROVED** [review]

**Pull request merged**